_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,116 @@
+
+<div style="display: block; width: inherit; height: 400px; text-align: center;">
+    <a href="/ITStrategy/assets/images/Battle_of_Borodino_1812.png" style="display: block">
+        <img src = "/ITStrategy/assets/images/Battle_of_Borodino_1812.png"
+        alt = "A painting of the Battle of Borodino during the invasion of Russia by France in the Napoleonic Wars"
+        style="width: 500px; height: inherit;"
+        />
+    </a>
+    <small >The Battle of Borodino</small>
+</div>
+<br/>
+
+## Looking at a possible alternative
+
+The goal of DevOps at its core is to shorten lead time by provide quick feedback loops to developers, business and operational staff. There are 3 main requirements that an organization should have to enable this.
+
+1. **Cross-functional teams**: Instead of having distinct teams with distinct functions, teams are built with all the required functions centred around products, applications or services. Functions are not solely delegated to one individual within the team, it is a group responsibility to ensure that compliance with items such as IT Security and Accessibility are met. This does not mean that teams such as IT Security, Accessibility and Quality Assurance cease to exist. They instead take on key enabling roles such as building and providing tooling such as automated testing tools such that compliance is ensured at build time, every time. These teams also serve a mentoring role, teaching developers on best practices in IT and application security, testing, data modelling and accessibility.
+
+2. **Automating compliance, testing, integration and deployment**: I mentioned this in the previous point but it is important I highlight this requirement specifically. DevOps relies on automating processes. In a modern technology organization, there is simply too much to be able to manually check in a timely manner with realistic resources. IT operational staff are shifted from the reactive role of putting out fires and instead are responsible for developing and improving tooling and pipelines that ensure compliance with every automated build and during development. For example, Enterprise Architecture would be responsible for building, developing and improving reusable components that, when used by a developer, will automatically implement reference architecture in applications. They are also responsible on mentoring and teaching developers through their wealth of knowledge on creating modular and decoupled applications.
```

Comment:
Do we use 'I' since these blog posts are technically from the team?  'I mentioned this ....'.  just curious.  Could be 'This is mentioned in the previous post but is important to highlight this requirement specifically'.

Also...'there is simply too much'.....what?  Maybe 'it is simply not possible to do manual checks in a timely manner..'

Reply:
There is too much checks to do manually is what I meant. Good point on the use of I